pallets/price-feed/src/lib.rs

```diff
@@ -76,6 +76,16 @@ pub mod pallet {
     pub type AssetFeeds<T: Config> =
         StorageMap<_, Blake2_128Concat, T::AssetId, FeedIdFor<T>, OptionQuery>;
 
+    #[pallet::storage]
+    /// (AssetId) -> AssetPricePair
+    ///
+    /// This storage stores the initial price pair for quote assets based on `SelfAssetId`
+    ///
+    /// * insert: adding a new asset with no price pair been set yet
+    /// * remove: an asset has its price pair, remove automatically
```

Comment:
Should we keep this around, since this is only a single entry?
removing it is probably not worth it

Reply:
Agree with keeping this without removing

I don't have a certain idea about this actually, this confused me a lot since submitting a new feed automatically while adding a new asset is not suitable

Reply:
yeah, chainlink feeds are kept independently from the assets, and the price pair pallet merely keeps a mapping.
you can also have a look at acala's chainlink integration https://github.com/AcalaNetwork/Acala/pull/1073/files